server/migrations/sql/20191025170040-create-tags-table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE tags(
```

Comment:
`tag` please -- aside from some of the first ones we made, all our table names are singular